docs/user-guide/ingress-path-matching.md

```diff
@@ -0,0 +1,142 @@
+# Ingress Path Matching
+
+## Regular Expression Support
+
+The ingress controller supports **case insensitive** regular expressions in the `spec.rules.http.paths.path` feild. 
+
+```
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
```

Comment:
is the `use-regex` annotation missing here?

Reply:
fixed!